scripts/base/frameworks/intel/cluster.zeek

```diff
@@ -17,11 +17,11 @@ global insert_indicator: event(item: Item) &is_used;
 const send_store_on_node_up = T &redef;
 
 # If this process is not a manager process, we don't want the full metadata.
-@if ( Cluster::local_node_type() != Cluster::MANAGER )
+@activate-if ( Cluster::local_node_type() != Cluster::MANAGER )
 redef have_full_data = F;
```

Comment:
> Seems that's not safe, as it depends on the previous value of have_full_data, which would be stepped on whether-or-not the cluster node is a non-manager.

I see the argument. I do really hope that in practice it would actually be safe to do because the intel cluster is confusing enough as is :-) Resolving.